src/room.cpp

```diff
@@ -32,6 +32,7 @@ void Room::Destroy() {
         server->Shutdown(300);
         RakNet::RakPeerInterface::DestroyInstance(server);
     }
+    members.clear();
```

Comment:
Because Destroy() is the only location the server can disconnect, thus Destroy == CloseConnection